ref/roadmap.md

```diff
@@ -8,8 +8,21 @@ Anbox Cloud follows a defined release cycle with frequent minor and patch releas
 
 **Patch releases** - A patch release for Anbox Cloud is released at the beginning of every month and includes Android and Chrome security updates alongside Anbox Cloud specific bug fixes.
 
+## Supported versions
+
+The current feature release of Anbox Cloud is 1.19.0. Anbox Cloud officially supports only the most recent release. Older releases are only supported for a short time after a new minor release was published.
+
+To ensure you receive latest security updates and bug fixes you should upgrade to a new release of Anbox Cloud shortly after it was released.
+
 ## Roadmap
 | Release | Target Date | What to expect| 
 |---------|-------------|---------------|
 | 1.19.1 | September 13 2023 | * Android security updates for September 2023<br/>* Bug fixes |
 | 1.19.2 | October 11 2023 | * Android security updates for October 2023<br/>* Bug fixes |
+| 1.20.0 | November 15 2023 |  |
+
+## Professional support
+
+If you are looking for additional support, find out about [Ubuntu Pro](https://ubuntu.com/support).
+
+Canonical can also provide [managed solutions](https://ubuntu.com/managed) for Anbox Cloud.
```

Comment:
Does that really fit in the roadmap document?

Reply:
I was moving it around to see if it is better to keep a separate topic for this or merge it. I didn't realise it got added to the commit. Thanks! I have reverted it and planning to keep it as is now.